docs/Users_Guide/scatter.rst

```diff
@@ -0,0 +1,227 @@
+************
+Scatter Plot
+************
+
+Description
+===========
+The scatter plot is useful for illustrating relationships between pairs of continuous variables.
+This plot was developed to support plotting MPR (matched pair) data from the MET point-stat tool.
+**NOTE:** This MET output data must first be reformatted into a format that can be read in by the scatter plot code.
+This reformatting was accomplished through the METdataio METreformat module. The reformatted data
+consists solely of MPR linetype data and all the column headers are labelled according to the
+MPR linetype column names specified in the MET User's Guide, Table 11.20 (Point-stat tool).
+In addition to selecting the two continuous variables to plot, the points in the scatter plot are colored based on
+their value.  These colors are determined by specifying a colormap in the user configuration file.
+
+Example
+=======
+
+Sample Data
+-----------
+
+The data is text output from the MET point-stat tool. The data is arranged in columnar format and reformatted by the
+METdataio METreformat module.
+
+The sample data for creating the example scatter plot is available in the
+`METplotpy <https://github.com/dtcenter/METplotpy>`_ repository, where the scatter plot tests are located:
+
+    *$METPLOTPY_BASE/metplotpy/test/scatter/reformatted_data_for_scatter.data*
+
+*$METPLOTPY_BASE* is the directory where the METplotpy code is saved:
+
+e.g.
+
+*/usr/path/to/METplotpy* if the source code was cloned or forked from the Github repository
+
+or
+
+*/usr/path/to/METplotpy-x.y.z* if the source code was downloaded as a zip or gzip'd tar file from the Release link of
+the Github repository.  The *x.y.z* is the release number.
+
+**NOTE**:
```

Comment:
Suggest using the note directive (.. note:) to be consistent with other notes in the documentation.

Reply:
done